package.json

```diff
@@ -83,11 +83,13 @@
     ]
   },
   "scripts": {
+    "__ci:prod": "PROD=1 DEV= esno ./workspace/scripts/index.ts ci --type specs -v",
```

Comment:
idk if we want to keep these -- who knows how long it'll be for vitest to support running one workspace